src/Utilities/Utilities.jl

```diff
@@ -543,6 +543,66 @@ function issubmodule(sub, mod)
     (sub === mod) || issubmodule(module_parent(sub), mod)
 end
 
+function single_command_line(args, kwargs, marker = "--")
+    result = ``
+    for arg in args
+        result = `$result $arg`
+    end
+    for (key, value) in kwargs
+        result = `$result $marker$key $value`
+    end
+    result
+end
+command_line(args...; kwargs...) =
+    single_command_line(args, kwargs)
+abbreviated_command_line(args...; kwargs...) =
+    single_command_line(args, kwargs, "-")
+
+exists(dir) = "Found $dir"
+not_exists(dir) = "Cannot find $dir"
+
+function info_dir(dir)
+    result = ispath(dir)
+    if result
+        dir |> exists |> info
+    else
+        dir |> not_exists |> info
+    end
+    result
+end
+
+function good_dir(dir)
+    if !info_dir(dir)
+        error("Please make sure it exists")
+    end
+    dir
+end
+
+function bad_dir(dir)
+    if info_dir(dir)
+        error("Please remove it")
+    end
+    dir
+end
+
+expand_dir(package, args...) =
+    joinpath( Pkg.dir(package) |> good_dir, args...)
+
+backup_dir(dir, pkgname) = if dir == nothing
+    Pkg.dir(pkgname)
+else
+    dir
+end |> good_dir
+
+path_check(dir) = if info_dir(dir)
+    ";" * dir
+    info("Adding to path")
+else
+    ""
+end
+
+add_to_path(dirs) = "Path" => string(ENV["Path"], map(path_check, dirs)...)
```

Comment:
Should be `PATH`, at least on linux. Somehow I have a feeling that windows would be case-insensitive anyway? Currently fails with

```
ERROR: KeyError: key "Path" not found
 in access_env(::Base.##339#340, ::String) at ./env.jl:45
 in add_to_path at /home/morten/Julia/pkg/v0.5/Documenter/src/Utilities/Utilities.jl:604 [inlined]
 in #genkeys#4(::Array{String,1}, ::Function, ::String, ::SubString{String}, ::SubString{String}) at /home/morten/Julia/pkg/v0.5/Documenter/src/Travis.jl:120
 in (::Documenter.Travis.#kw##genkeys)(::Array{Any,1}, ::Documenter.Travis.#genkeys, ::String, ::SubString{String}, ::SubString{String}) at ./<missing>:0
 in #generate#21(::Void, ::String, ::Bool, ::Array{String,1}, ::Function, ::String) at /home/morten/Julia/pkg/v0.5/Documenter/src/Documenter.jl:602
 in generate(::String) at /home/morten/Julia/pkg/v0.5/Documenter/src/Documenter.jl:569
```